src/pages/ServiceDetails/ServiceInfo/IstioObjectDetails/VirtualServiceDetail.tsx

```diff
@@ -87,7 +87,11 @@ class VirtualServiceDetail extends React.Component<VirtualServiceProps> {
           undefined
         )}
         {virtualService.gateways && virtualService.gateways.length > 0 ? (
-          <DetailObject name="Gateways" detail={virtualService.gateways} />
+          <DetailObject
+            name="Gateways"
+            detail={virtualService.gateways}
+            href={`/namespaces/istio-system/istio/gateways`}
```

Comment:
At the point, the URL has to be built dynamically. Check this [out](https://istio.io/docs/reference/config/istio.networking.v1alpha3/#VirtualService) so see how the gateways can be specified in that field.